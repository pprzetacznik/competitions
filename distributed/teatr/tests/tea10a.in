99999999 0 -1 -1 1000000 8100225648422384477
