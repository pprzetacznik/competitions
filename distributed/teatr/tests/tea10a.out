2500001134547955
